### DISCUSSION TITLE
How do I execute existing bash scripts

### DISCUSSION THREAD

**Discussion by thesobercoder** [nushell/nushell#3810] [Q&A] [answered]
Loving nushell so far. We have quite a number of bash scripts from work that I often need to execute but I have been pulling up the bash shell for that. Is there a way I can do it from the nushell itself?

I'm on Windows 10, using Git bash.

**Comment by kubouch** [comment, marked as answer]
I think there are two options:
1. run `bash script.bash` 
2. put a "shebang" line at the first line of the script: `#!/usr/bin/env bash`, make sure it's executable: `chmod +x script.bash` and then just execute it: `./script.bash`.

This is based on Linux. Not sure if 2. would work on Git bash on Windows, but at least the first one should work.
